f21/ps.html

```diff
@@ -1700,31 +1724,25 @@ <h2><u> Lecture / Lab 13</u></h2>
 
 
 
-<li > <b>Due Date: 12 December</b>  <b style="float:right">Reading: <a href="https://www.cplusplus.com/files/tutorial.pdf">Cplusplus Tutorial</a> & <a href="lab13.html">Lab 13</a></b><br>
+<li > <b>Due Date: 9 December</b>  <b style="float:right">Reading: <a href="https://www.cplusplus.com/files/tutorial.pdf">Cplusplus Tutorial</a> & <a href="lab13.html">Lab 13</a></b><br>
 
-	<p> Write a <b>C++ program</b>  that asks the user for the starting amount, and prints out the yearly balance of a savings account, assuming 5% interest for the first 5 years and 10% interest after that, until the amount has increased by more than $1000.
+	<p> Write a <b>C++ program</b>  that asks the user for their monthly budget on food and coffee and ask the user how much they spend on coffee in a week and print out how much they have left to spend every week after purchasing coffee.
 
 <p> A sample run:
-<pre style ="background-color:lavender;width:400px">
-  Please enter the starting amount: 1000
-  Year 1  1050.00
-  Year 2  1102.50
-  Year 3  1157.62
-  Year 4  1215.51
-  Year 5  1276.28
-  Year 6  1403.91
-  Year 7  1544.30
-  Year 8  1698.73
-  Year 9  1868.60
-  Year 10  2055.46
+<pre style ="background-color:lavender;width:450px">
+  Enter your monthly budget for food and coffee: 400
+  How much are you spending in a week for coffee: 40.50
+  Budget left after week 1	359.5
+  Budget left after week 2	319
+  Budget left after week 3	278.5
+  Budget left after week 4	238
 </pre>
 
-<p> Note: the autograder is expecting each line to begin with "Year " and the corresponding number for the year.
-<p><b>IMPORTANT:</b> For grading purposes, set the precision to two decimal places. To do so, add <tt>#include &lt;iomanip&gt;</tt> at the top of the file, and <tt>cout << fixed << setprecision(2); </tt>.
+<p> Note: the autograder is expecting each line to begin with "Week" and the corresponding number for the week
```

Comment:
I'm not sure what this means, in the sample output each line begins with "Budget left..." what you expect the students to do should be consistent with the sample output, which is it? Or am I not understanding the note?

Reply:
Sorry about that. You don't need the line to start with "week" or "budget left after week...".

Reply:
i removed that line. i tested it without that line on the autograder.